doc/sphinx/language/core/conversion.rst

```diff
@@ -3,11 +3,75 @@
 Conversion rules
 ----------------
 
-|CiC| has conversion rules that can be used to determine if two
-terms are equal by definition, or :term:`convertible`.
-Conversion rules consist of reduction rules and expansion rules.
-See :ref:`applyingconversionrules`,
-which describes tactics that apply these conversion rules.
+Conversion rules convert terms to other terms that are equivalent by
+definition under the rules of |CiC|.  Conversion rules can be used, for example,
+to determine if two terms are equal by definition, or :term:`convertible`, by
+converting both terms to a normal form, then verifying they are syntactically
+equal (ignoring differences in the names of bound variables
+:term:`alpha-convertibility <alpha-convertible>`).
+
+Conversion rules can be reductions or expansions.
+Reductions convert terms to something that is incrementally closer to its
+normal form.  For example, :term:`zeta-reduction` removes
+:n:`let @ident := @term__1 in @term__2` constructs from a term by replacing
+:n:`@ident` with :n:`@term__1` wherever :n:`@ident` appears in :n:`@term__2`.
+The resulting term may be longer or shorter than the original.
+
+.. coqtop:: all
+
+   Eval cbv zeta in let i := 1 in i + i.
+
+Expansions are reductions applied in the opposite direction,
+for example expanding `2 + 2` to `let i := 2 in i + i`.  While applying
+reductions (except for eta-reduction) gives a unique result , the associated
```

Comment:
Eta reduction gives a unique result, but the result is not necessarily of a type that's convertible with the type of the expanded form.